examples/batch/v1beta1/jobqueue.yaml

```diff
@@ -0,0 +1,335 @@
+apiVersion: batch.aws.upbound.io/v1beta1
+kind: JobQueue
+metadata:
+  annotations:
+    meta.upbound.io/example-id: batch/v1beta1/jobqueue
+  labels:
+    testing.upbound.io/example-name: test_queue
+  name: test-queue
+spec:
+  forProvider:
+    computeEnvironmentOrder:
+      - computeEnvironmentSelector:
+          matchLabels:
+            testing.upbound.io/example-name: sample
+        order: 1
+    priority: 1
+    region: eu-west-1
+    state: ENABLED
+  providerConfigRef:
+    name: aws-provider-config-test     
```

Comment:
We use the default provider config in uptest; could you please remove the above?